code/game/jobs/job_exp.dm

```
@@ -0,0 +1,211 @@
+
+// Defines
+
+#define EXP_TYPE_CREW			"Crew"
+#define EXP_TYPE_COMMAND		"Command"
+#define EXP_TYPE_ENGINEERING	"Engineering"
+#define EXP_TYPE_SECURITY		"Security"
+#define EXP_TYPE_SILICON		"Silicon"
+#define EXP_TYPE_SERVICE		"Service"
+#define EXP_TYPE_MEDICAL		"Medical"
+#define EXP_TYPE_SCIENCE		"Science"
+#define EXP_TYPE_SUPPLY			"Supply"
+#define EXP_TYPE_SPECIAL		"Special"
+#define EXP_TYPE_GHOST			"Ghost"
+#define EXP_TYPE_EXEMPT			"Exempt"
+
+// Globals, Defines, Etc
+
+
+
+
+// Admin Verbs
+
+
+/client/proc/cmd_admin_check_player_exp()	//Allows admins to determine who the newer players are.
+	set category = "Admin"
+	set name = "Check Player Playtime"
+	if(!check_rights(R_ADMIN))
+		return
+	var/msg = "<html><head><title>Playtime Report</title></head><body>Playtime:<BR><UL>"
+	for(var/client/C in clients)
+		msg += "<LI> - [key_name_admin(C)]: <A href='?_src_=holder;getplaytimewindow=\ref[C.mob]'>" + C.get_exp_general() + "</a></LI>"
+	msg += "</UL></BODY></HTML>"
+	src << browse(msg, "window=Player_playtime_check")
+
+
+/datum/admins/proc/cmd_show_exp_panel(var/client/C)
+	if (!C)
+		return
```

This also needs feedback, instead of failing silently.